db/migrate/20220721182316_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.text :text
+      t.timestamps
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] The reference should be to `author` while mapping it to the `user` table. Please kindly adjust to meet ERD diagram requirements.

Hint
`t.references :author, foreign_key: { to_table: 'users' }`